README.md

```diff
@@ -0,0 +1 @@
+I added more text here.
```

Comment:
Hey @saron, I think you should reconsider the spelling of added. I think it needs more ds. Ask @kjaymiller about how he spells words.